internal/proxy/root_handler.go

```diff
@@ -85,7 +85,7 @@ func writeError(w http.ResponseWriter, storage string, msg string, code int, log
 func handleJSONErrorResponse(log *logrus.Entry, w http.ResponseWriter, code int, err error) {
 	log.Error(err)
 	w.WriteHeader(code)
```

Comment:
`web.JSONErrorResponse` is writing the code so we don't need to do it here. We might see `superfluous response.writeheader call` errors in the logs.

Reply:
fixed